bindings/python/scripts/idyntree-model-view-meshcat.py.in

```diff
@@ -0,0 +1,37 @@
+#!@Python3_EXECUTABLE@
```

Comment:
This will hardcode the python executable used at build time in this script. Do you have any specific reason for which you do not want to use the usual Python3 shebang?
```suggestion
#!/usr/bin/env python3
```

Reply:
I was thinking if a `virtual enviroment` is used

Reply:
For that using `/usr/bin/env python3` should work fine.

Reply:
Ok! perfect!

Reply:
Hi @traversaro, https://github.com/robotology/idyntree/pull/961/commits/3c9e2436f7f2901d1b0516f9b6b6715204949118 should implement what you suggested